kitsune/kbadge/signals.py

```diff
@@ -31,7 +31,7 @@ def my_callback(sender, **kwargs):
 def _signal_with_docs(args, doc):
     # FIXME - this fixes the docstring, but not the provided arguments
     # so the API docs look weird.
-    signal = Signal(providing_args=args)
+    signal = Signal(args)
```

Comment:
I believe this needs to be `Signal` and the args should be either dropped or added in the comments/docstring

Reply:
Got it.